src/quicklist.c

```diff
@@ -131,6 +146,7 @@ quicklist *quicklistCreate(void) {
     quicklist->head = quicklist->tail = NULL;
     quicklist->len = 0;
     quicklist->count = 0;
+    quicklist->alloc_size = sizeof(*quicklist);
```

Comment:
it could be that the quicklist struct, although size padded, won't exactly fit a jemalloc bin. and thus the sizeof won't give the right value.

Reply:
Done.